Code/Framework/AzFramework/AzFramework/Quality/QualityCVarGroup.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Memory/Memory_fwd.h>
+#include <AzCore/Console/IConsole.h>
+#include <AzCore/Console/ConsoleFunctor.h>
+#include <AzCore/std/string/string.h>
+#include <AzCore/std/string/string_view.h>
+
+#include <AzFramework/Quality/QualitySystemBus.h>
+
+namespace AzFramework
+{
+    // QualityCVarGroup wraps a QualityLevel CVAR for a quality group that, when set,
+    // iterates over all settings in the group and performs console commands to
+    // apply the settings for the requested quality level.
+    // The general format of a quality group entry in the Settings Registry is:
+    // {
+    //     "O3DE" : {
+    //         "Quality" : {
+    //             "Groups" : {
+    //                 "<group CVAR>" : {
+    //                      "Description" : "<optional description>",
+    //                      "Levels" : [ "<quality level 0>", "<quality level n>" ],
+    //                      "Default" : "<default quality level>",
+    //                      "Settings" : {
+    //                          "<setting CVAR>" : [<level 0 value>, <level n value> ]
+    //                      }
+    //                  }
+    //             }
+    //         }
+    //     }
+    // }
+    //    
+    // QualityCVarGroup only creates a CVAR for the quality group itself, it does not
+    // create CVARs for any entries in the quality groups "Settings" object.
+    // Quality levels are assumed to be ordered low to high as in the example below.
+    //
+    // Example:
+    // A q_shadows group CVAR exists with 4 levels (low=0,medium=1,high=2,veryhigh=3)
+    // When the level of q_shadows is set to 2 LoadQualityLevel will apply all
+    // console settings defined in the SettingsRegistry for that
+    // group at level 2 (high).
+    //

```

Comment:
nit: Large multi-line comment blocks should use the C-style block comment /* */ instead to protect against any potential final escape `\` character at the end of the line